Source/Extensions/Blazorise.SignaturePad/SignaturePad.razor.cs

```diff
@@ -0,0 +1,283 @@
+    /// </summary>
+    [JSInvokable]
+    public async Task NotifyBeginStroke()
+    {
+        await BeginStroke.InvokeAsync();
+    }
+
+    /// <summary>
+    /// Gets the string representation of the <see cref="ImageType"/>.
+    /// </summary>
+    /// <param name="imageType">Image type value.</param>
+    /// <returns>String representation of the <see cref="ImageType"/>.</returns>
+    private static string ToImageTypeString( SignaturePadImageType imageType ) => imageType switch
+    {
+        SignaturePadImageType.Jpeg => "jpeg",
+        SignaturePadImageType.Svg => "svg",
+        SignaturePadImageType.Png => "png",
+        _ => "png",
+    };
+
+    #endregion
+
+    #region Properties
+
+    /// <summary>
+    /// Reference to the object that should be accessed through JSInterop.
+    /// </summary>
+    protected DotNetObjectReference<SignaturePad> DotNetObjectRef { get; private set; }
+
+    /// <summary>
+    /// Gets or sets the <see cref="JSSignaturePadModule"/> instance.
+    /// </summary>
+    protected JSSignaturePadModule JSModule { get; private set; }
+
+    /// <summary>
+    /// Gets or sets the JS runtime.
+    /// </summary>
+    [Inject] private IJSRuntime JSRuntime { get; set; }
+
+    /// <summary>
+    /// Gets or sets the version provider.
+    /// </summary>
+    [Inject] private IVersionProvider VersionProvider { get; set; }
+
+    ///<summary>
+    /// Gets or sets value for the signature pad.
+    /// </summary>
+    [Parameter] public byte[] Value { get; set; }
+
+    /// <summary>
+    /// Gets or sets the event that is triggered when the signature pad value changes. The event provides the new signature as a byte array.
+    /// </summary>
+    [Parameter] public EventCallback<byte[]> ValueChanged { get; set; }
+
+    /// <summary>
+    /// Gets or sets the event that is triggered when a stroke ends on the signature pad. The event provides the signature pad's current image data as a PNG-encoded Data URL.
+    /// </summary>
+    [Parameter] public EventCallback EndStroke { get; set; }
+
+    /// <summary>
+    /// Gets or sets the event that is triggered when a new stroke begins on the signature pad. The event provides information about the starting point of the stroke.
+    /// </summary>
+    [Parameter] public EventCallback BeginStroke { get; set; }
+
+    /// <summary>
+    /// The radius of a single dot. Also the width of the start of a mark.
+    /// </summary>
+    [Parameter] public double DotSize { get; set; }
+
+    /// <summary>
+    /// The minimum width of a line.
+    /// </summary>
+    /// <value>The minimum width.</value>
+    [Parameter] public double MinLineWidth { get; set; }
+
+    /// <summary>
+    /// The maximum width of a line.
+    /// </summary>
+    /// <value>The maximum width.</value>
+    [Parameter] public double MaxLineWidth { get; set; }
+
+    /// <summary>
+    /// The time in milliseconds to throttle drawing. Set to 0 to turn off throttling.
```

Comment:
How would you feel about doing these type of time based api parameters with a timespan instead? That gives the flexibility of users doing `Timespan.FromMinutes(1);`
And we can just do `TimeSpan.Milliseconds`?